analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocation.cs

```diff
@@ -341,6 +362,95 @@ public void Instance_InstanceMethodCallClearsField()
                 new SymbolicValue().WithConstraint(ObjectConstraint.NotNull));
         }
 
+        [TestMethod]
+        public void Invocation_StaticMethodCallClearsField()
```

Comment:
Rename & let's move it up, near the other `Invocation_StaticMethodCall_Xxxx`
```suggestion
        public void Invocation_StaticMethodCall_ClearsField()
```

Reply:
I moved it after the "Instance" tests and renamed the "Instance" tests also.